kuma/urls.py

```diff
@@ -81,6 +81,8 @@
     url(r'^miel$',
         handler500,
         name='users.honeypot'),
+    # We use our own views for setting language in cookies. But to just align with django, set it like this.
+    url(r'^i18n/setlang/', core_views.set_language),
```

Comment:
@This should be in the ``LANGUAGE_URL_IGNORED_PATHS`` setting. Or do you mean to serve it at ``/{locale}/i18n/setlang``? What else did you want at ``/i18n/``?